pkg/network/cache/infocache.go

```diff
@@ -68,6 +75,11 @@ type PodInterfaceCacheStore interface {
 	Remove() error
 }
 
+type VMNetworkingInfraInterfacesStore interface {
+	Read(iface string) (*networkdriver.VIF, error)
+	Write(iface string, cacheInterface *networkdriver.VIF) error
```

Comment:
the implemenation of Read/Write has a mix of iface and ifaceName arg naming, let's use `ifaceName` wich is exactly what we pass.

Reply:
Done.